docs/operator-manual/webhook.md

```diff
@@ -12,6 +12,35 @@ a Git webhook for GitHub, but the same process should be applicable to other pro
     the same. A hook event for a push to branch `x` will trigger a refresh for an app pointing at the same repo with
     `targetRevision: refs/tags/x`.
 
+### Webhook Ingress
+
+The applicationset-controller is responsible for answering to webhook calls so a separate ingress has to be setup for it.
+
+Example Ingress:
+```
+# base/argocd-webhook-ingress.yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: argo-cd-webhook
+spec:
+  rules:
+    - host: cd.apps.argoproj.io
+      http:
+        paths:
+          - path: /api/webhook
+            pathType: Prefix
+            backend:
+              service:
+                name: argocd-applicationset-controller
+                port:
+                  number: 7000
+  tls:
+    - hosts:
+        - cd.apps.argoproj.io
+      secretName: argocd-secret
+```
```

Comment:
Link to the ingress documentation. The underlying user infrastructure may diverge too much from the example app.

```suggestion
The api path `/api/webhook` of the `argocd-applicationset-controller` service on the `webhook` named port must be configured as part of your [ingress](./ingress.md).
```

Reply:
Your sentence could fit nicely into a `!!! note` or similar. A doc that describe the different webhooks would also be nice, since I wasn't aware the argocd server also handled webhooks from reading the parts I were.